lib/mercury/fake.rb

```diff
@@ -17,15 +17,15 @@
 # broken sockets, etc.
 class Mercury
   class Fake
-    def self.install(rspec_context, domain=:default)
+    def self.install(rspec_context, domain = :default)
```

Comment:
Now that I think about it, this is of little concern. Most of the time we should be using keyword arguments rather than optional arguments anyway.